clients/admin-ui/src/features/privacy-requests/privacy-requests.slice.ts

```diff
@@ -359,6 +359,11 @@ export const privacyRequestApi = createApi({
         body: params,
       }),
     }),
+    getActiveStorage: build.query<any, void>({
```

Comment:
Are you able to generate the API schema response for these endpoints using [http://localhost:8080/openapi.json](http://localhost:8080/openapi.json)? This will give you a strongly typed type instead of using the `any`.

Reply:
Being covered in https://github.com/ethyca/fides/issues/2455